src/main.js

```diff
@@ -7,23 +7,24 @@ import ShowMoreView from './view/showMore';
 import FilmsAmountView from './view/filmsAmount';
 import PopupView from './view/popup';
 import {createFilm} from './mock/film';
-import {getRandomInteger, renderElement, RenderPosition} from './utils';
+import {getRandomInteger} from './utils/common';
+import {renderElement, RenderPosition, remove} from './utils/render';
 
 const header = document.querySelector('.header');
 const main = document.querySelector('.main');
-const CART_AMOUNT = 6;
-const CART_START = 5;
-const CART_ADDED = 5;
-const CART_AMOUNT_EXTRA = 2;
+const CARD_AMOUNT = 6;
+const CARD_START = 5;
+const CARD_ADDED = 5;
+const CARD_AMOUNT_EXTRA = 2;
 const body = document.body;
 
-const filmsArray = new Array(CART_AMOUNT).fill().map(createFilm);
+const filmsArray = new Array(CARD_AMOUNT).fill().map(createFilm);
```

Comment:
в принципе Array не обязательно указывать, достаточно просто Films

```suggestion
const films = new Array(CARD_AMOUNT).fill().map(createFilm);
```